{
  try {
    h();
  }
 catch (  RuntimeException e) {
  }
catch (  Exception e) {
  }
}
